src/coreclr/jit/ssabuilder.cpp

```diff
@@ -853,6 +853,46 @@ void SsaBuilder::RenameDef(GenTreeOp* asgNode, BasicBlock* block)
     }
 }
 
+//------------------------------------------------------------------------
+// RenameCall: Rename a local or memory definition generated by a GT_CALL node.
```

Comment:
I would really look into folding `RenameCall` into `RenameDef`.

E. g. `RenameDef` calls `AddDefToHandlerPhis`, `RenameCall` doesn't (presumably it needs to). There is also the memory renaming case. It would be best for all that logic to be in one method.

Reply:
done.